diff --git a/node_modules/@jsonforms/examples/src/index.ts b/node_modules/@jsonforms/examples/src/index.ts
index 2194d16..e18dac1 100644
--- a/node_modules/@jsonforms/examples/src/index.ts
+++ b/node_modules/@jsonforms/examples/src/index.ts
@@ -59,7 +59,6 @@ import * as scope from './examples/scope';
 import * as listWithDetail from './examples/list-with-detail';
 import * as listWithDetailRegistered from './examples/list-with-detail-registered';
 import * as object from './examples/object';
-import * as i18n from './examples/i18n';
 import * as issue_1948 from './examples/1948';
 import * as issue_1996 from './examples/1996';
 import * as issue_1169 from './examples/1169';
@@ -125,7 +124,6 @@ export {
   listWithDetail,
   listWithDetailRegistered,
   object,
-  i18n,
   issue_1169,
   issue_1220,
   issue_1253,
